web/src/features/exchanges/ExchangeArrow.tsx

```diff
@@ -19,11 +21,17 @@ function ExchangeArrow({ data, viewportWidth, viewportHeight, map }: ExchangeArr
   const colorBlindModeEnabled = false; // TODO: FIX https://linear.app/electricitymaps/issue/ELE-1384/set-up-colorblind-mode-that-changes-co2-scale
   const absFlow = Math.abs(data.netFlow ?? 0);
   const { co2intensity, lonlat, netFlow, rotation, key } = data;
-
+  const setIsMoving = useSetAtom(mapMovingAtom);
   if (!lonlat) {
     return null;
   }
 
+  useEffect(() => {
+    const cancelWheel = (event: WheelEvent) => event.preventDefault();
+    document.body.addEventListener('wheel', cancelWheel, { passive: false });
+    return () => document.body.removeEventListener('wheel', cancelWheel);
+  }, []);
```

Comment:
I assume there is no way to skip the `preventDefault()` and make this a passive event listener in our current implementation?

Reply:
Unfortunately passive tells the browser it can still do it's thing without considering the JS so it will lead to a double zoom effect with the whole page being zoomed at the same time as the map.

Reply:
The alternative solution would be like we used to handle this with our own listeners for all of the things

Reply:
Let's go with this for now and then add a task about evaluating the performance benefits (if any) of switching to custom event listeners.